rep-0150.rst

```diff
@@ -0,0 +1,210 @@
+REP: 150
+Title: ROS Melodic and Newer Metapackages
+Author: Tully Foote <tfoote@willowgarage.com>, Chris Lalancette <clalancette@openrobotics.org>
```

Comment:
Willow Garage email.

Reply:
Yeah, I wasn't quite sure what to do about that.  This file is largely a copy of REP-142, with some small differences, so I ended up keeping this email address the same.  @tfoote , should I just change this to your @openrobotics.org address?

Reply:
Yes please. The willowgarage address doesn't get delivered anymore.

Reply:
Sure.  Done in 7a5191e